proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpMetricsUtils.java

```diff
@@ -105,6 +113,29 @@ private static List<ReportPoint> fromOtlpRequest(
     return wfPoints;
   }
 
+  /*MONIT-30703: adding application & system.name tags to a metric*/
+  @VisibleForTesting
+  public static void updateAttrsListForOtelMetrics(
+      Resource resource, String otlpMetric, List<KeyValue> attrList) {
+    if (OtlpTraceUtils.isOtelMetric(otlpMetric)) {
+      String appName =
```

Comment:
Optional: lines 121-128 and lines 129-136 are very similar. You could extract a method here.

Reply:
done